tensorflow/python/keras/activations.py

```diff
@@ -68,19 +68,48 @@ def softmax(x, axis=-1):
 
 @keras_export('keras.activations.elu')
 def elu(x, alpha=1.0):
-  """Exponential linear unit.
+  """Exponential Linear Unit.
+
+  The exponential linear unit (ELU) with `alpha > 0` is:
+  `x` if `x > 0` and
+  `alpha * (exp(x) - 1)` if `x < 0`
+  The ELU hyperparameter `alpha` controls the value to which an
+  ELU saturates for negative net inputs. ELUs diminish the
+  vanishing gradient effect.
+
+  ELUs have negative values which pushes the mean of the activations
+  closer to zero.
+  Mean activations that are closer to zero enable faster learning as they
+  bring the gradient closer to the natural gradient.
+  ELUs saturate to a negative value when the argument gets smaller.
+  Saturation means a small derivative which decreases the variation
+  and the information that is propagated to the next layer.
+
+  Example Usage:
+
+  ```python3
+  model = models.Sequential()
```

Comment:
Can this snippet be changed to DocTest format so it can be run and tested? See https://www.tensorflow.org/community/contribute/docs_ref